analytics/views/stats.py

```diff
@@ -258,6 +264,11 @@ def get_chart_data(
     remote_realm_id: Optional[int] = None,
     server: Optional["RemoteZulipServer"] = None,
```

Comment:
I think the test suites are failing here due to the forward referencing used here. I'm not sure why forward referencing is being used here, or why the imports are conditional in the first place. Might be better to just import the classes regardless?

Reply:
It's because the `zilencer` project isn't included in self-hosted releases.